core/utils.class.inc.php

```diff
@@ -167,9 +167,9 @@ static public function Log($iPriority, $sMessage)
 		}
 	}
 
-	static public function MockLog($MockedLogger)
+	static public function MockLog($oMockedLogger)
     {
```

Comment:
Mixed types of indetation, tabs vs spaces.

PS most IDE's can fix this automatically for the whole file.